data/json/items/containers.json

```diff
@@ -177,7 +177,7 @@
       {
         "pocket_type": "CONTAINER",
         "watertight": false,
-        "max_contains_volume": "2 L",
+        "max_contains_volume": "2500 ml",
```

Comment:
The can that bread comes in is larger than a two liter of coca cola?

Reply:
Well, let's see... something like 4.25 inches tall, I'm estimating a diameter of 2 inches... call it 875 mL, at least for cans from the original company.

Reply:
This is the powder bag for bread flour. I guess I should rename it to `bag_paper_powder` instead of `bag_paper_powder_small`...